cmd/shield/help/create-job

```diff
@@ -50,7 +53,8 @@ USAGE: @G{shield} create-job --tenant @Y{TENANT} [OPTIONS]
       @Y{--target}    "Important Files" \
       @Y{--store}     "Amazon S3"       \
       @Y{--schedule}  "daily 4am"       \
-      @Y{--retain}    14d
+      @Y{--retain}    14d               \
+      @Y{--retries}    2
```

Comment:
This one has 2 vs 3 or 7, we should be consistent.

Reply:
I'll change this to 3.